client/src/modules/CrossCheck/__tests__/AddCriteriaForCrossCheck.test.tsx

```diff
@@ -19,6 +19,14 @@ describe('AddCriteriaForCrossCheck', () => {
 
   test('should call addCriteria when "Add new criteria" button was clicked', () => {
     render(<AddCriteriaForCrossCheck onCreate={addCriteria} />);
+    const selectCriteriaType = screen.getByRole('combobox') as HTMLSelectElement;
+    fireEvent.mouseDown(selectCriteriaType);
+    const optionTitle = screen.getByTestId('Title') as HTMLOptionElement;
```

Comment:
Queries are generics that accept returned element type. So if you really want to provide returned element type, it's better to do it that way to avoid typecast:  
  ```suggestion
      const selectCriteriaType = screen.getByRole<HTMLSelectElement>('combobox');
      fireEvent.mouseDown(selectCriteriaType);
      const optionTitle = screen.getByTestId<HTMLOptionElement>('Title');
```
But in this particular case I am not sure that we need it here at all :)

Reply:
Maksim, thanks a lot for review! I've made necessary changes